web/app/js/components/MetricsTable.jsx

```diff
@@ -251,6 +277,7 @@ MetricsTable.propTypes = {
   metrics: PropTypes.arrayOf(processedMetricsPropType),
   resource: PropTypes.string.isRequired,
   selectedNamespace: PropTypes.string.isRequired,
+  jaeger: PropTypes.string,
```

Comment:
There is a bit of confusion here as `grafana` is marked as `isRequired` while `jaeger` is not. As both of these values are passed as empty strings when the arg is empty, This should not cause any problems!


The change would be to remove `isRequired` for `grafana` and update the tests as they expect grafana field to be there always.

Reply:
raised https://github.com/linkerd/linkerd2/issues/4349 to track